frontend/lobby/team.tsx

```diff
@@ -8,27 +8,27 @@ import { Scenario, OtherTeamMember } from '../../common/models';
 import { AnyAction } from '../global/actions';
 import { RpcClientConnectionStatus } from '../rpc-client/rpc-client-actions';
 
+import * as saltine from './images/saltine.svg';
+
+
 
 class OnlineIndicator extends React.PureComponent<{when: boolean}> {
     public render() {
         if (this.props.when) {
-            return <span className="online-indicator">(online)</span>
+            return <span className="online-indicator"></span>
```

Comment:
It's probably fine for this app, but normally you should keep the text ("(online)") there for non-visual users, and hide it with CSS.

Reply:
Ah, good point. We might as well add it in. I used [this W3 guideline](https://www.w3.org/WAI/tutorials/fundamentals/hiding/) to do it.